.devcontainer/devcontainer.json

```diff
@@ -3,10 +3,28 @@
   "build": {
     "dockerfile": "Dockerfile"
   },
-  "remoteUser": "root"
-  /*,
-  update below source folder to your local sdk-repos folder
-  "mounts": [
-    "source=/home/rc/repos/tmp/sdk-repos,target=/workspaces/sdk-repos,type=bind,consistency=cached"
-  ]*/
+  "postCreateCommand": "npm install",
+  "remoteUser": "root",
+  "remoteEnv": { 
+    "PATH": "${containerEnv:PATH}:${containerWorkspaceFolder}/node_modules/.bin" 
+  },
+  "features": {
+    "ghcr.io/devcontainers/features/node:1": {
```

Comment:
More of a question but what does this do on top of already using the node base image?

Reply:
Very good point :)
it does nothin I'll remove it.
I started with a different image, where this was necessary, then moved to the node-typescript base image, which uses node 20 already:

➜  ~ docker run -it --rm mcr.microsoft.com/devcontainers/typescript-node:20-bookworm /bin/bash
root ➜ / $ node --version
v20.13.1
root ➜ / $ npm --version
10.5.2

removing int